sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Processor/TestProcessor.cs

```diff
@@ -227,6 +297,19 @@ private void UploadBuffer(string uri, string buffer, string fileRelativePath)
             blobClient.Upload(new BinaryData(bufferBytes), overwrite: true);
             _logger.Info($"Uploaded buffer to {fileRelativePath}");
         }
+
+        private void UploadBlobFile(string uri, string fileRelativePath, string filePath)
+        {
+            string cloudFilePath = GetCloudFilePath(uri, fileRelativePath);
+            BlobClient blobClient = new(new Uri(cloudFilePath));
+            // Upload filePath to Blob
+            blobClient.Upload(filePath, overwrite: true);
+
+            //byte[] bufferBytes = File.ReadAllBytes(fileRelativePath);
+            //blobClient.Upload(new BinaryData(bufferBytes), overwrite: true);
+            _logger.Info($"Uploaded file {filePath} to {fileRelativePath}");
```

Comment:
Won't this log too many times, not sure if it would be useful for debugging customer issues.

Reply:
1 per attachment should be fine to debug in case of any issue in attachment upload mismatch.

Reply:
Same as previous, can we move to debug?